src/test/rgw/bucket_notification/README.rst

```diff
@@ -0,0 +1,43 @@
+============================
+ Bucket Notification tests
+============================
+
+The tests use the Nose test framework. To get started, ensure you have
+the ``virtualenv`` software installed; e.g. on Debian/Ubuntu::
+
+        sudo apt-get install python-virtualenv
+
+and then run::
+
+        ./bootstrap
```

Comment:
bootstrap required for sudo privileges.  why is this needed, given that we run in a virtualenv?

Reply:
It would install the required packages

Reply:
if it installed them locally for the user or inside the virtualenv it DOES NOT NEED SUDO